src/gvars.c

```diff
@@ -825,6 +881,34 @@ static Obj FuncIsReadOnlyGVar (
     return IsReadOnlyGVar(GVarName(CSTR_STRING(name))) ? True : False;
 }
 
+/****************************************************************************
+**
+*F  IsConstantGVar( <gvar> ) . . . . . . return if a variable is a constant
```

Comment:
Hrmf, somehow GitHub decided that the comments I was collecting for a review were all to be suddenly published as comments instead, taking away from me the possibility to edit them beforehand... I was about to add the following (or possibly, replace those comments completely with it) regarding the comment formatting: "OK, I see now that these files you are touching are particularly bad when it comes to the comment formatting (and grammar...). So it probably makes more sense to go over them separately later on, and to "clean up" those comments, and/or to remove parts of them."